crates/optimism/hardforks/src/lib.rs

```diff
@@ -96,6 +97,7 @@ pub static OP_MAINNET_HARDFORKS: LazyLock<ChainHardforks> = LazyLock::new(|| {
         (OpHardfork::Holocene.boxed(), ForkCondition::Timestamp(1736445601)),
         (EthereumHardfork::Prague.boxed(), ForkCondition::Timestamp(1746806401)),
         (OpHardfork::Isthmus.boxed(), ForkCondition::Timestamp(1746806401)),
+        (OpHardfork::Jovian.boxed(), ForkCondition::Timestamp(u64::MAX)), /* TODO: Update timestamp when Jovian is planned */
```

Comment:
we cant include this just yet, and should just either comment this out or remove from the statics entirely

Reply:
comment out is a good idea